sdk/src/main/java/ch/admin/bag/covidcertificate/eval/decoder/CertificateDecoder.kt

```diff
@@ -41,7 +41,7 @@ object CertificateDecoder {
 
 		val bagdgc = CborService.decode(cbor, qrCodeData) ?: return DecodeState.ERROR(Error(ErrorCodes.DECODE_CBOR))
 
-		bagdgc.certType = CertTypeService.decode(bagdgc.euDGC)
+		bagdgc.certType = CertTypeService.decode(bagdgc.euDGC!!)
```

Comment:
In the long run are: are we guaranteed that this is non null? What if a DccLight was decoded?

Reply:
The dcclight handling will be added in the next feature. for this refactoring i would not handle it.